### ISSUE TITLE
Bot respond with url without http(s) for follow request

### ISSUE THREAD

**Issue by brbrr** [Automattic/wp-telegram-bot#18]
In channel chat when following new blog:
`follow http://testblog.wordpress.com`

Bot will respond with:
`Now following testblog.wordpress.com`

**Comment by markryall** [comment]
Yes I think we should always just completely remove the protocol from the interaction - XMPP subsc don't care about it and neither should the bot.

The post urls are always going to be https:// even if you specified http:// when setting up the follow
